nuxt.config.js

```diff
@@ -239,7 +239,7 @@ export default {
   apollo: {
     clientConfigs: {
       ...defineApolloConfig(),
-      subsquid: toApolloEndpoint(process.env.SUBSQUID_ENDPOINT),
+      subsquid: toApolloEndpoint(process.env.SUBSQUID_ENDPOINT || ''),
```

Comment:
just noticed that won't work for newcomers?
let's tune it to the best, w @vikiival approval

```suggestion
      subsquid: toApolloEndpoint(process.env.SUBSQUID_ENDPOINT || 'https://app.gc.subsquid.io/beta/rubick/004/graphql'),
```